app/webpack.production.js

```diff
@@ -4,12 +4,13 @@ const common = require('./webpack.common')
 
 const webpack = require('webpack')
 const merge = require('webpack-merge')
-const ExtractTextPlugin = require('extract-text-webpack-plugin')
```

Comment:
Same as above, this doesn't seem to have been ported to webpack 4 based on https://github.com/webpack-contrib/extract-text-webpack-plugin/issues/701 and https://github.com/webpack-contrib/extract-text-webpack-plugin/issues/782 so I've churned away to something else that's compatible.